src/components/new-safe/steps/Step2/index.tsx

```diff
@@ -48,12 +49,15 @@ const CreateSafeStep2 = ({
     },
   })
 
-  const { register, handleSubmit, control, watch } = formMethods
+  const { register, handleSubmit, control, watch, formState } = formMethods
 
   const allFormData = watch()
 
   const { fields: ownerFields, append: appendOwner, remove: removeOwner } = useFieldArray({ control, name: 'owners' })
 
+  const isValid = Object.keys(formState.errors).length === 0
```

Comment:
Does `formState.isValid` not work? If it's `true` from the start, I'd suggest we call this `hasErrors` or something instead.

Reply:
I copied this from the old form where it had a comment saying: `// do not use formState.isValid because names can be empty`

Reply:
Can you try `isValid` again as we updated RHF? If it doesn't work, it'd be nice to keep the comment.

Reply:
Changed it to `formState.isValid` and it works fine, thanks!